lib/public/Files/IMimeTypeLoader.php

```diff
@@ -47,4 +47,14 @@ public function exists(string $mimetype): bool;
 	 * @since 8.2.0
 	 */
 	public function reset(): void;
+
+	/**
+	 * Update filecache mimetype based on file extension
+	 *
+	 * @param string $ext
+	 * @param int $mimeTypeId
+	 * @return int
+	 * @since 8.2.0
```

Comment:
Same here you added this only now the OCP interface, even if it was there in OC for some while.
Adding it to OCP means its public API now with fixed method signature while before it was just in OC and could change at any point.

```suggestion
	 * @since 32.0.0
```

Reply:
Ah, fair enough ! :)